app/frontend/tests/unit/composables/documentTemplate.spec.js

```diff
@@ -113,15 +113,15 @@ describe('documentTemplate.js', () => {
 
   describe('readFile', () => {
     it('should return a valid file', async () => {
-      // no data
-      const mockInputElement = document.createElement('input');
-      mockInputElement.type = 'file';
-      const file = await readFile(mockInputElement);
```

Comment:
This was failing because readFile doesn't take a DOM element as input. Not sure how it ever passed.